lib/onc_certification_g10_test_kit.rb

```diff
@@ -140,6 +141,9 @@ def self.well_known_route_handler
 
     group from: 'multi_patient_api'
 
+    group from: 'multi_patient_api_stu2',
+          id: 'Group06'
```

Comment:
This is on the wrong group. It got added in another PR that was merged, so you can just remove it.